models/commons/src/address/address-draft/presets/change-history-data/us-address.ts

```diff
@@ -0,0 +1,7 @@
+import { TAddressDraftBuilder } from '../../../types';
+import * as AddressDraft from '../../index';
+
+const usAddress = (): TAddressDraftBuilder =>
+  AddressDraft.random().country('US');
```

Comment:
Just curious, is `empty()` useful for you here or do you want the additional randomized data?

Reply:
The reason I ask, is it might make your life testing these easier with `toMatchInlineSnapshot` (I'm a fan, just a suggestion), but I don't know what your requirements are